samples/certs/Makefile

```diff
@@ -23,6 +24,12 @@ CITADEL_CN ?= Intermediate CA
 CITADEL_SAN_DNS ?= localhost
 # Additional variables are defined in %/intermediate.conf target below.
 
+#------------------------------------------------------------------------
+# variables: workload certs: eg VM
+WORKLOAD_DAYS ?= 365
```

Comment:
Too much ? Too little ? 

Technically the agent should be able to renew - if we set it to 1 year nobody will 
test renewal, and 1 year from now we'll have broken VMs if someone uses this.

Let's set it to 1 day - so 'stability' clusters and release tests catch rotation issues.

Note that we are discussing 2 modes ( and I think the agent is supporting both ), 
one will use a very long lived provisioning cert - maybe as long as the root cert - 
and the old one where the cert should last an outage but be self-rotated.